src/library/Uno.Material/Styles/Controls/NavigationView_SplitView.xaml

```diff
@@ -89,7 +89,7 @@
 																	 Storyboard.TargetProperty="TranslateX"
 																	 From="{Binding TemplateSettings.OpenPaneLength, RelativeSource={RelativeSource Mode=TemplatedParent}}"
 																	 To="0"
-																	 Duration="0:0:0.10" />
```

Comment:
animations in the SplitView I'm not too worried, they're probably very specific to that control and don't need to be using the global duration. We use the  Material_AnimationDuration for Opacity animation everywhere but here it's a translation. I would leave the 0.0.25 or use a new variable name